From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Matti Lehtimaki <matti.lehtimaki@gmail.com>
Date: Sun, 21 Dec 2025 03:14:06 +0200
Subject: [PATCH] (hybris) Use linkerconfig from
 /usr/libexec/droid-hybris/system/bin

Needed for allowing /usr/libexec/droid-hybris/system/ paths to be used
from namespaces.

Change-Id: I2fd0be43c8d2c899a108db687dd4204e477f91de
---
 init/builtins.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/init/builtins.cpp b/init/builtins.cpp
index 3e1f16f2caccbcaf1fa65eeab47812ef7bf566be..2b27e4e5aeae0d5bd018492f47e52fb149604e87 100644
--- a/init/builtins.cpp
+++ b/init/builtins.cpp
@@ -1202,7 +1202,7 @@ static Result<void> do_mark_post_data(const BuiltinArguments& args) {
 }
 
 static Result<void> GenerateLinkerConfiguration() {
-    const char* linkerconfig_binary = "/apex/com.android.runtime/bin/linkerconfig";
+    const char* linkerconfig_binary = "/usr/libexec/droid-hybris/system/bin/linkerconfig";
     const char* linkerconfig_target = "/linkerconfig";
     const char* arguments[] = {linkerconfig_binary, "--target", linkerconfig_target};
 
